src/metabase/util.clj

```diff
@@ -94,6 +94,21 @@
              (re-matches #"[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?"
                          (str/lower-case s)))))
 
+(defn state?
+  "Is `s` a state string?"
+  ^Boolean [^String s]
+  (contains? #{"alabama" "alaska" "arizona" "arkansas" "california" "colorado" "connecticut" "delaware"
```

Comment:
Is this too USA-centric? E.g., what if I'm a Brazilian user and my `State` field contains "acre" "bahia" "goias" "roraima" "sao paulo" etc.? We have users in many countries, many of which are made up of states (Brazil, Australia, Germany, India …).

Reply:
do we have any UI bits that can show maps of other countries? I raised this issue in the description with a possibility of looking for some config file for states we could check. But do we have existing or planned UI bits to take advantage of it? I'm also not sure where `:type/State` would be used besides the USA map but i certainly want to make sure I haven't violated any assumptions.

Reply:
You can upload your own custom maps, but AFAIK we don't have a way to label a map as the official "map of states" for your instance. Without a feature like that marking non-US states as `type/State` might lead to a worse experience if it often displays as a blank map of the US. I haven't tested it, but that would be my guess.

That said, it'd be sweet to bundle more maps with Metabase. Maybe when we do that it'd make sense to have separate types for each country's states/regions? That does inflate the type system a lot though.

Reply:
ok. it sounds like non-us-state recognition is a future endeavor then? perhaps in the future we can have some kind of `:type.State/US`, `:type.State/Brazil` or some similar notion but that day is not today.

Reply:
Greeeaaat point, Paul. I lost sight of what marked a field as `type/State` actually does currently. 🤦‍♂️